gen

```diff
@@ -1,11 +1,13 @@
-#! /bin/sh
+#!/bin/sh
 
 set -eu
 
 export GOBIN=$PWD/bin
 export PATH=$GOBIN:$PATH
 
-go install ./vendor/github.com/golang/protobuf/protoc-gen-go
-go install ./vendor/github.com/spf13/cobra/cobra
+if [ `which protoc-gen-go 2>/dev/null` == "" ] ; then
+    echo "try: go install github.com/golang/protobuf/protoc-gen-go"
+    exit 1
+fi
```

Comment:
I think this would be better off as

```suggestion
if ! which protoc-gen-go >/dev/null 2>&1; then
    echo "try: go install github.com/golang/protobuf/protoc-gen-go"
    exit 1
fi
```